coordinator/core/core.go

```diff
@@ -187,6 +189,12 @@ func NewCore(dnsNames []string, qv quote.Validator, qi quote.Issuer, sealer seal
 		return nil, err
 	}
 
+	// TODO(katexochen): not sure if this is the right place for this
+	servers, err := c.data.getTimeServers()
```

Comment:
Yes, the idea was to only set a trusted time if the state contains a trusted time server list. Previous in the core generation, the time is set to an untrusted time.
We could check whether a manifest was set at this point, but since trusted time is optional in the manifest, this wouldn't give any additional information to our expectation to find a list of trusted time servers in the store.

Reply:
I see, fair enough. Please add a log warning if we're running with untrusted time. Thanks!